openedx/features/caliper_tracking/utils.py

```diff
@@ -74,3 +74,23 @@ def get_team_url_from_team_id(referer, team_id):
         team_id=team_id
     )
     return object_link
+
+
+def get_certificate_url(user_id, course_id):
+    """
+    This function takes user_id and course_id as parameters
+    makes a reverse url for certificate generated by that user.
+
+    :return: A URI of required certificate.
+    """
+    certificate_uri = '{lms_url}{certificate_link}'.format(
+        lms_url=settings.LMS_ROOT_URL,
+        certificate_link=str(reverse(
+            'certificates:html_view',
```

Comment:
Certificates are sometimes PDF files. Are we sure that this reverse link will work in that case as well?

Reply:
In open-edx certificates are only viewed using its html page, it can be printed from there, but its not in pdf format.
In-short, hopefully it will work.

Reply:
Alright.